src/lib/config.js

```diff
@@ -57,7 +57,7 @@ class Config {
       }
     }
     // eslint-disable-next-line no-console
-    console.log('Config loaded: %s', util.inspect(this))
+    console.log('Initializes database connection to :', RC.DATABASE.SCHEMA, 'database.')
```

Comment:
why is a console.log still here. please remove it. it wont print

Reply:
Will remove.

Reply:
There are a number of places console.log is used in the code. Have commented the code out for now.